lib/fluent/plugin/in_windows_eventlog2.rb

```diff
@@ -108,7 +116,11 @@ def start
       super
 
       @chs.each do |ch, read_existing_events|
-        subscribe_channel(ch, read_existing_events)
+        if (read_all_channels)
+          subscribe_channel(ch, read_existing_events) rescue ''
```

Comment:
We shouldn't use Pokemon exception handling.
I'll implement checking mechanism for un-subscribable channels.

Reply:
I'd implemented it in https://github.com/fluent-plugins-nursery/winevt_c/pull/22.